python/rpdk/java/templates/init/guided_aws/StubDeleteHandler.java

```diff
@@ -0,0 +1,71 @@
+package {{ package_name }};
+
+import software.amazon.cloudformation.exceptions.ResourceNotFoundException;
+import software.amazon.cloudformation.proxy.AmazonWebServicesClientProxy;
+import software.amazon.cloudformation.proxy.Logger;
+import software.amazon.cloudformation.proxy.ProgressEvent;
+import software.amazon.cloudformation.proxy.ProxyClient;
+import software.amazon.cloudformation.proxy.ResourceHandlerRequest;
+
+public class {{ operation }}Handler extends BaseHandlerStd {
+    private Logger logger;
+
+    protected ProgressEvent<ResourceModel, CallbackContext> handleRequest(
+        final AmazonWebServicesClientProxy proxy,
+        final ResourceHandlerRequest<ResourceModel> request,
+        final CallbackContext callbackContext,
+        final ProxyClient<ServiceSdkClient> proxyClient,
+        final Logger logger) {
+
+        this.logger = logger;
+
+        final ResourceModel model = request.getDesiredResourceState();
+
+        // TODO: Adjust Progress Chain according to your implementation
+
+        return proxy.initiate("Service-Name::Delete-Custom-Resource", proxyClient, model, callbackContext)
+            .request(Translator::translateToDeleteRequest) // construct a body of a request
+            .call(this::deleteResource) // make an api call
+            .stabilize(this::stabilizeOnDelete) // stabilize is describing the resource until it is in a certain status
+            .success(); // success if stabilized
```

Comment:
This call chain only represents one generic way to stabilize, and it needs to be explicitly noted as such (which is why I suggest commenting it out).  I would recommend using a real working example (like, take Cloudwatch logs or SES or some other simple resource) rather than fake code that doesn't work.

Also, I am confused by `"Service-Name::Delete-Custom-Resource"`.  I would have no idea what to put there

Reply:
well it seems to be a bit overkill bc the open sourced repo doesnt use this pattern but boiler plate will? if so then we should refactor existing code of cloudwatch. But I like the idea